subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/DataBaseSchemaUtil.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.performance.results;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+class DataBaseSchemaUtil {
+    private static final String[] REMOVED_COLUMNS = new String[] {
+        "heapUsageBytes",
+        "totalHeapUsageBytes",
+        "maxHeapUsageBytes",
+        "maxUncollectedHeapBytes",
+        "maxCommittedHeapBytes",
+        "compileTotalTime",
+        "gcTotalTime",
+        "executionTime",
+        "configurationTime"
+    };
+
+    static void removeOutdatedColumnsFromTestDB(Connection connection, Statement statement) throws SQLException {
```

Comment:
I have made that change.

Looks fine on CI: https://builds.gradle.org/viewLog.html?buildId=2323275 (failures are performance issues that we current have on master)

I would now merge this and then prepare another small PR that removes the columns for merging after the next release.